src/xtgeo/well/blocked_well.py

```diff
@@ -168,7 +170,16 @@ def copy(self):
 
         return newbw
 
-    def to_roxar(self, *args, **kwargs):
+    def to_roxar(
+        self,
+        project: str,
```

Comment:
Should probably be type Any since `project` may be a string or a "magic" variable